db/migrate/20221018194832_create_comment.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComment < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+
+      t.timestamps
+    end
```

Comment:
- [ ] there should be a reference to Author and post in the comment table in the form of a foreign key.